spec/app_spec.rb

```diff
@@ -62,5 +62,14 @@
 
       expect(last_response.body).to include(email)
     end
+
+    it 'has a logout link back to root' do
+      get '/auth/result', code: code
+
+      doc = Nokogiri::HTML(last_response.body)
+
+      logout_link = doc.at('a[href="/"]')
+      expect(logout_link.text.strip).to eq('Log out')
```

Comment:
is there anything else we can look for to ensure that visiting this url makes the user be logged out?